app/models/website.rb

```diff
@@ -1,9 +1,10 @@
+# frozen_string_literal: true
 class Website < ApplicationRecord
+  before_validation :add_protocol_to_website 
+  validates :url, url: true
```

Comment:
Group different types of statements (callbacks, validations, associations, etc) together, and separate them by empty lines.